src/gatsby-node.ts

```diff
@@ -24,25 +24,44 @@ function validateConfigOptions (configOptions: ConfigOptions): void {
 export function sourceNodes (sourceNodesArgs: SourceNodesArgs, configOptions: ConfigOptions): Promise<void[]> {
   validateConfigOptions(configOptions)
 
-  return Promise.all([createPublicBusinessUnitInfoNode(sourceNodesArgs, configOptions)])
+  return Promise.all([createPublicBusinessUnitInfoNode(sourceNodesArgs, configOptions),
+    createPublicBusinessUnitReview(sourceNodesArgs, configOptions)])
 }
 
-function createPublicBusinessUnitInfoNode (sourceNodesArgs: SourceNodesArgs, configOptions: ConfigOptions): Promise<void> {
+async function createPublicBusinessUnitInfoNode (sourceNodesArgs: SourceNodesArgs, configOptions: ConfigOptions): Promise<void> {
```

Comment:
if we are using async-awaits, then we need to use try catch blocks to when the await fails we can throw an error with a descriptive message